### PR TITLE
Update for grade history

### PR DESCRIPTION
**Pull request by luismorenolopera**
Update for grade history.

- add the earned_graded_override field to PersistentSubsectionGradeOverrideHistory model.
- save the earned_graded_override and comments fields in PersistentSubsectionGradeOverrideHistory model.
- the [model](https://github.com/edx/edx-platform/commit/8234208cbba5244ef9051eaa594837bff0d6898c
) that stores the change history is updated, which in higher versions does not exist.

This PR partially depends on [this](https://github.com/Zollege/frontend-app-gradebook/pull/6) , for visual effects only.

**ironwood.master:**
image

**Before:**
image

**After:**

image

### CODE REVIEW

File: lms/djangoapps/grades/models.py [hunk 1]
```diff
@@ -761,6 +764,7 @@ class Meta(object):
     user = models.ForeignKey(User, blank=True, null=True)
     comments = models.CharField(max_length=300, blank=True, null=True)
     created = models.DateTimeField(auto_now_add=True, db_index=True)
+    earned_graded_override = models.FloatField(null=True, blank=True)
```
Comment (jfavellar90) [inline lms/djangoapps/grades/models.py:767]: is this to add support for grades overrides? is not it supported in Ironwood out of the box? @luismorenolopera

Comment (luismorenolopera) [reply lms/djangoapps/grades/models.py:767]: That value is only called in a log, but it was not stored.

**Comment by morenol** [comment]
It is worth to mention that every grade override done before this change will not display the earned_graded_override and the comments since they are not being stored in the Ironwood version and also that every update to the grades done with this change is possible that will not be displayed once we migrate to Juniper since the way that Juniper stores that information is different and requires several changes.

**Comment by jfavellar90** [comment]
@morenol @luismorenolopera how critical is the fact that we likely won't be able to see grade updates after Juniper migration? could we create any data migration to guarantee that grade updates are displayed on Juniper if the migration is executed? or is this an issue that we are putting off and we will need to tackle it when Juniper migration for Zollege comes?

**Comment by morenol** [comment]
> @morenol @luismorenolopera how critical is the fact that we likely won't be able to see grade updates after Juniper migration? could we create any data migration to guarantee that grade updates are displayed on Juniper if the migration is executed? or is this an issue that we are putting off and we will need to tackle it when Juniper migration for Zollege comes?

Yes, that is something that we should do once we take Juniper migration

**Comment by felipemontoya** [comment]
@luismorenolopera if I understand correctly, once we run this (https://github.com/edx/edx-platform/commit/8234208cbba5244ef9051eaa594837bff0d6898c#diff-e5e2d0ea9c7b847332651a176827bbaaR16) migration, the whole model will be removed. So this means there will not be a clash of the migrations.

Am I missing something?

**Review by felipemontoya — APPROVED** [review]

**Comment by luismorenolopera** [comment]
@felipemontoya correct, in the future to save this data [django-simple-history](https://django-simple-history.readthedocs.io/en/latest/) will be used

**Pull request merged**